user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -740,7 +740,7 @@
   },
   "PMC": {
     "NameOverride": false,
-    "ForceCustomWaves": true,

```

Comment:
why we enabled this ? should conflict with Donuts isn't it ?

Reply:
We shouldn't